snapcraft/internal/sources/__init__.py

```diff
@@ -84,13 +93,19 @@
 from ._subversion import Subversion  # noqa
 from ._tar import Tar                # noqa
 from ._zip import Zip                # noqa
+from . import errors
+
+if sys.version_info < (3, 6):
```

Comment:
I took the example from [PR#940](https://github.com/snapcore/snapcraft/pull/940) [snapcraft/file_utils.py](https://github.com/snapcore/snapcraft/pull/940/files#diff-8672afd5aee283a8c90cf4487fb51689): In python 3.6 sha3 support is upstreamed in hashlib.